### PR TITLE
Comments now have css

### PR DESCRIPTION
**Pull request by gabrielbursztein**
Css in comments and Books API

### CODE REVIEW

File: app/assets/stylesheets/comments.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  border-top: 1px solid $opacity-light;
+
+  .comments-container {
+    border-top: 1px solid $opacity-light;
+
+    .new-comment {
+      width: inherit;
+      margin-top: 30px;
+
+      h4 {
```
Comment (santiagovidal) [inline app/assets/stylesheets/comments.scss:21]: Maybe it's too much work to change it now, but keep in mind for the future that nesting with more than 2 levels is not a very good practice.

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
+      <div class="book-author"><%= book.author %></div>
+      <br>
+      <div class="book-year-request">
+        <h4><%= book.year %></h4> 
+        <% if !current_user.requested?(book) %>
+          <span id='<%= book.id %>'>
+            <%= link_to 'Request', requests_path(book_id: book.id), method: :post, remote: true, class: "request" %>
+          </span>
+        <% else %>
+          <span class="<% if current_user.request_status(book) == "denied"%>denied bold<% else %>bold dark-opacity big-20<% end %>"><%= current_user.request_status(book) %></span>
```
Comment (santiagovidal) [inline app/views/books/show.html.erb]: Use an inline `if`.

Also, the class `bold` is always used, so you can exclude it from the conditional.

File: app/views/books/show.html.erb [hunk 3]
```diff
@@ -1,30 +1,41 @@
...
-    <table>
+  <div class='comments-container'>
+    <h3>Comments</h3>
       <% book.comments.each do |comment|%>
-        <tr>
-          <td><%= comment.description %></td>
-          <td><%= comment.rate %></td>
-        </tr>
+        <div class="comment">
+          <h5>Gabriel - </h5><%= comment.rate %> <br>
```
Comment (santiagovidal) [inline app/views/books/show.html.erb]: Gabriel?

File: app/views/books/show.html.erb [hunk 4]
```diff
@@ -1,30 +1,41 @@
...
+        </div>
+        <div class="line-separator"></div>
       <% end %>
-    </table>
+    <div id="add-comment"></div>
+    <div class="new-comment">
+      <%= form_for :comment, url: book_comments_path(book), remote: true do |f| %>
+          <h4>Add a comment - Rate:</h4>
+          <%= f.number_field :rate, max:5, min:1 %> <br>
+          <%= f.text_area :description, placeholder:"Comment", id:"text_area" %> <br>
```
Comment (santiagovidal) [inline app/views/books/show.html.erb]: Don't use ids to set CSS properties. Use classes for that.

File: app/views/comments/create.js.erb [hunk 5]
```diff
@@ -1,2 +1,2 @@
 
-$('.comments').append('<%= @comment.rate%><%= @comment.description%>')
+$('#add-comment').append('<div class="comment"><h5>Gabriel - </h5><%= @comment.rate %> <br><%= @comment.description %></div><div class="line-separator"></div>');
```
Comment (santiagovidal) [inline app/views/comments/create.js.erb]: Gabriel?

**Review by santiagovidal — CHANGES_REQUESTED** [review]

File: app/controllers/books_controller.rb [hunk 6]
```diff
@@ -1,9 +1,22 @@
 class BooksController < ApplicationController
-    
     helper_method :book
+    before_action :authenticate_user!
 
     def index
       @books = Book.all
+      respond_to do |format| 
+        format.html
+        format.json {render json: {status:'SUCCESS', message:'All books',
```
Comment (MaicolBen) [inline app/controllers/books_controller.rb]: Remove `status:'SUCCESS', message:'All books'`

File: app/controllers/books_controller.rb [hunk 7]
```diff
@@ -1,9 +1,22 @@
...
+      data: @books}, status: :ok}
+      end  
+    end
+
+    def show
+      respond_to do |format| 
+        format.html
+        format.json {render json: {status:'SUCCESS',
```
Comment (MaicolBen) [inline app/controllers/books_controller.rb]: Same as above

File: app/views/books/show.html.erb (hunk 2 above)
Comment (MaicolBen) [reply app/views/books/show.html.erb]: I prefer using `if` but in multiple lines, because it's too long

**Review by MaicolBen — COMMENTED** [review]
Remove `b/test.sqlite3`

**Review by santiagovidal — APPROVED** [review]

**Pull request merged**